online.rst

```diff
@@ -22,32 +22,39 @@ To log in, please click the link titled: “Already a member? Login”, then ent
 
 HMS is managed by the Software team.
 
+
+Discord - https://wiki.nottinghack.org.uk/Discord
+`````````````````````````````````````````````````
+Discord is an instant messaging platform which we to allow teams and members to communicate. Your discord account needs to be linked to your Hackspace Management System account in order to see the majority of channels.
+
+We have a separate policy on using Discord, which covers some basic netiquette. This can be found as `Appendix C on our rules site <https://rules.nottinghack.org.uk/en/latest/discord.html>`_.
+
+Our Discord also has a number of commands which interact in some way with the hackspace. For example, you can check the temperature of various areas by sending "!temp", or search the Wiki with "!wiki <search term>"
+
 Wiki - https://wiki.nottinghack.org.uk
 ``````````````````````````````
 The Nottinghack Wiki is where we store information around projects and equipment, as well as organisation details for big events such as Makerfaire and EMF. As a member please feel free to start a user page to hold build logs of your projects, or when you're collaborating with other members on projects and need a place to store information.
 
 You can use your HMS account details to log into the wiki and make edits.
 
-Planet Nottinghack - https://planet.nottinghack.org.uk
-``````````````````````````````````````````````
-If you blog about Hackspace relevant stuff, projects you've made here or the subject of making and hacking as a whole, you’d be most welcome to contribute your feed to Planet Nottinghack. Many members run their own blogs, and Planet Nottinghack uses RSS to collect them all into one place.  If you would like to be included on this, please open a pull request against the GitHub repository.  If you’re not sure how to do this then ask on the Google Group.
+Rules - https://rules.nottinghack.org.uk
+````````````````````````````````````````
+
+You should be aware of all Nottingham Hackspace rules and our complaints policy. These are broken up into sections, much like this guide.
+
```

Comment:
Not sure if RST cares, but there's an extra newline here, and no newline after the title in the next section (Members Guide).